controllers/tasks.js

```diff
@@ -142,23 +143,44 @@ const updateTaskStatus = async (req, res, next) => {
   try {
     const taskId = req.params.id;
     const { dev } = req.query;
-    const { id: userId } = req.userData;
+    const { id: userId, username } = req.userData;
     const task = await tasks.fetchSelfTask(taskId, userId);
 
     if (task.taskNotFound) return res.boom.notFound("Task doesn't exist");
     if (task.notAssignedToYou) return res.boom.forbidden("This task is not assigned to you");
     if (task.taskData.status === "VERIFIED")
       return res.boom.forbidden("Status cannot be updated. Please contact admin.");
 
-    await tasks.updateTask(req.body, taskId);
+    const taskLog = {
+      type: "task",
+      meta: {
+        userId,
+        taskId,
+        username,
+      },
+      body: [],
+    };
+
+    if (req.body.status) {
+      taskLog.body.push(`${username} changed task status to ${req.body.status}`);
+    }
```

Comment:
if in the frontend I want to check the value of status, then I wouldn't be able to check it right? I think we should send the status as well with this. what do you think?